tdrs-backend/tdpservice/parsers/schema_defs/header.py

```diff
@@ -21,19 +21,19 @@
             validators.matches('HEADER'),
         ]),
         Field(item="4", name='year', type='number', startIndex=6, endIndex=10, required=True, validators=[
-            validators.between(2000, 2099)
+            validators.isInLimits(2000, 2099)
         ]),
         Field(item="5", name='quarter', type='string', startIndex=10, endIndex=11, required=True, validators=[
             validators.oneOf(['1', '2', '3', '4'])
         ]),
         Field(item="6", name='type', type='string', startIndex=11, endIndex=12, required=True, validators=[
             validators.oneOf(['A', 'C', 'G', 'S'])
         ]),
-        Field(item="1", name='state_fips', type='string', startIndex=12, endIndex=14, required=True, validators=[
-            validators.between(0, 99)
+        Field(item="1", name='state_fips', type='number', startIndex=12, endIndex=14, required=True, validators=[
+            validators.isInLimits(0, 99)
         ]),
-        Field(item="3", name='tribe_code', type='string', startIndex=14, endIndex=17, required=False, validators=[
-            validators.between(0, 999)
+        Field(item="3", name='tribe_code', type='number', startIndex=14, endIndex=17, required=False, validators=[
```

Comment:
similar suggestion to the above: tribe_code should be a string type. and can be `000, 999` or `   `